data-changes/location-codes.md

```diff
@@ -0,0 +1,93 @@
+# Location codes change
+
+This document provides details of a correction we have applied in 2021 to correctly represent location code of GeoNet strong motion (SM) stations.
+The change has been applied to data and metadata which have been corrected during 2021. 
+
+You might be impacted by this change if you:
+- have a local copy of our data and/or metadata that is not up to date;
+- are using our data and metadata services while we apply the change;
+- are using a derived data product that has a mismatch in the metadata 
+
+
+## Background
+Location codes naming conventions used for GeoNet seismic stations are documented [on our _delta_ metadata repository](https://github.com/GeoNet/delta/tree/main/docs).
+Based on this nomenclature convention:
+- all strong motion sensors have a location code `2?`;
+- when a sensor is moved more than 1 m (but less than 200m), its location code will change accordingly (e.g. from `20` to `21`).
+
+This convention aligns with [naming requirements for Seismohraph Stations](http://www.isc.ac.uk/registries/registration/) established by the International Seismological Centre (ISC).
+
+For GeoNet seismic sensors, the convention was established in September 2019.
+
+### Change description
+After the convention was confirmed, we conducted an audit on all equipment changes across the GeoNet National Strong motion sensor network. We found that a number of SM sensors were moved to a new location, without having their location code changed accordingly.
+
+We are now retrofitting our data archive and our metadata records, so that this change is properly represented.
+
+For those SM stations, location codes that were previously recorded as `20` are now being changed to `21`.
+
+Due to the complexity of this work, there have been for a period of time a mismatch between the metadata (that were correctly reporting 21) and the data in the archive. This has now been fixed.
+
+### Affected data
+
+Data and metadata that are affected by this change are:
+- strong motion metadata (distributed via delta, FDSN)
+- strong motion miniseed data (distributed via FDSN)
+- strong motion processed data (distributed via the Strong Motion Tool and archive)
+- seismic data picks ID and magnitude ID (distributed via FDSN) might have metadata not correctly associated
```

Comment:
what are seismic data picks ID and magnitude ID?

Reply:
that was a comment from @salichon , I understand those are metadata available in the quakeml ? if there is a better description of that I'd really appreciate suggestions!

Reply:
No idea at all. Just curious to know what those IDs are. The only ID I'm aware of is the event public ID.